cli/build.rs

```diff
@@ -111,4 +116,20 @@ fn main() {
     &main_module_name,
   )
   .expect("Failed to create snapshot");
+
+  set_binary_metadata();
 }
+
+#[cfg(target_os = "windows")]
+fn set_binary_metadata() {
+  let mut res = winres::WindowsResource::new();
```

Comment:
I really dig that we can pull this in as a cargo dep, rather than figuring out how to get `rc.exe` installed on every user's machine. I wish we could do that with our ninja/gn/python build dependencies too.

Reply:
Read through the implementation a couple of days ago, it doesn't bring it's rc.exe. It has to be in PATH which is the case if you have the Windows SDK.

Definitively agree that it is is clean and convenient how this works out tho!

Reply:
This is why my original implementation did not treat winres compilation errors as fatal - so users without rc.exe can keep building Deno on their machines. I guess we can always improve this in the future, perhaps by fetching rc.exe on-demand, if the need arises.